docs/skill_development/user-interaction/conversational-context.md

```diff
@@ -0,0 +1,161 @@
+---
+description: >-
+  A Skill can add context to the Intent Parser to create more natural
+  interaction with Neon.
+---
+
+# Conversational Context
```

Comment:
Just a note to validate this works since I don't think I've ever used it. Specifically, this might not work with multiple users